sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1767,6 +1767,11 @@ object SQLConf {
         "with String")
     .booleanConf
     .createWithDefault(false)
+
+  val DEFAULT_V2_CATALOG = buildConf("spark.sql.default.catalog")
+      .doc("Name of the default v2 catalog, used when an catalog is not identified in queries")
```

Comment:
nit. `an catalog` -> `a catalog`.

Reply:
as we discussed in the DS v2 meeting, we should clearly point out which places this default catalog is used. View/Function resolution definitely doesn't use this default catalog for now.

Reply:
I agree, we need to make clear choices and document where this is used.